cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,86 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+import { confirmationMessage } from '../plugins/alertMessages';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
+  let user;
+  let followButtonSelector;
+  let followButtonText;
+  let unfollowButtonSelector;
+  let unfollowButtonText;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      cy.visit('/#/register');
+      signUpPage.fillForm(user.username, user.email, user.password);
+      signUpPage.submitSignUpForm();
+
+      cy.get('.swal-button--confirm').contains(confirmationMessage).click();
+      homePage.clickNewArticle();
+
+      const title = faker.lorem.words(2);
+      const description = faker.lorem.words(2);
+      const content = faker.lorem.words(2);
+      const tags = 'tag1';
+
+      articlePage.fillTitle(title);
+      articlePage.fillDescription(description);
+      articlePage.fillContent(content);
+      articlePage.fillTags(tags);
+      articlePage.submitArticle();
 
+      homePage.clickSettings();
+      cy.scrollTo('bottom');
+      settingsPage.logOutSettings();
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
+  beforeEach(() => {
+    const nameNextUser = faker.name.firstName();
+    const emailNextUser = faker.internet.email();
+    const passwordNextUser = faker.internet.password();
+
+    cy.visit('/#/register');
+    signUpPage.fillForm(nameNextUser, emailNextUser, passwordNextUser);
+    signUpPage.submitSignUpForm();
+    cy.get('.swal-button--confirm').contains(confirmationMessage).click();
+
+    cy.visit('/#/');
```

Comment:
Direct navigation to the URL '/#/' is used. Try to follow the user journey through the application's UI instead.

Reply:
Done